index.html

```diff
@@ -2,9 +2,150 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Frontend Project Juni</title>
+    <link rel="stylesheet" href="styles.css">
 </head>
 <body>
+<header class="bg-gradient angle-shape-bottom">
+    <div class="container-center container-center__header">
+        <nav class="navbar">
+            <ul>
+                <li><a href="#projects">Case Studies</a></li>
+                <li><a href="#services">Services</a></li>
+                <li><a href="#contact">Contact Us</a></li>
+            </ul>
+        </nav>
+        <div class="header-content">
+            <h1>Your website is an investment in the success of your business.</h1>
+            <p>We are here to help you succeed</p>
+            <a class="btn-styling">let's work together</a>
```

Comment:
Een anchor tag zonder href is niet de bedoeling! Je had hier beter een button element van kunnen maken